test/evp_kdf_test.c

```diff
@@ -27,26 +27,37 @@ static EVP_KDF_CTX *get_kdfbyname(const char *name)
     return kctx;
 }
 
+static OSSL_PARAM *construct_tls1_prf_params(const char *digest, const char *secret,
+    const char *seed)
+{
+    OSSL_PARAM *params = OPENSSL_malloc(sizeof(OSSL_PARAM) * 4);
+    OSSL_PARAM *p = params;
+
+    *p++ = OSSL_PARAM_construct_utf8_string(OSSL_KDF_PARAM_DIGEST,
+                                            (char*)digest, strlen(digest));
```

Comment:
fix the spacing on the casts e.g (char *)

Reply:
Done